modules/tinymce/src/plugins/table/main/ts/selection/TableSelection.ts

```diff
@@ -6,28 +6,33 @@
  */
 
 import { CellOpSelection, Selections, TableSelection } from '@ephox/darwin';
-import { Arr, Optionals } from '@ephox/katamari';
+import { Arr, Fun, Optionals } from '@ephox/katamari';
 import { TableLookup } from '@ephox/snooker';
 import { Attribute, Compare, SelectorFind, SugarElement, SugarElements, SugarNode } from '@ephox/sugar';
 
 import { ephemera } from './Ephemera';
 
-const getSelectionStartCellFallback = (start: SugarElement<Node>) =>
-  TableLookup.table(start).bind((table) =>
+const getSelectionCellFallback = (element: SugarElement<Node>, takeLast: boolean) =>
+  TableLookup.table(element).bind((table) =>
     TableSelection.retrieve(table, ephemera.firstSelectedSelector)
-  ).fold(() => start, (cells) => cells[0]);
+  ).fold(
+    Fun.constant(element),
+    (cells) => takeLast ? cells[cells.length - 1] : cells[0]);
```

Comment:
This formatting is incorrect, as the `);` should be on a newline. However, this shouldn't matter when this is reverted, as it it should go back to a single line.

Reply:
It shouldn't be rewerted, since there should be situations where it's used, now when I've added the correct variables. Not quite sure what the situation is, but it should be there now.